CHANGELOG.md

```diff
@@ -197,6 +198,7 @@
 - PR #5233 Remove experimental namespace used during libcudf++ refactor
 - PR #5213 Documentation enhancements to `cudf` python APIs
 - PR #5251 Fix more mispellings in cpp comments and strings
+- PR #5254 Deprecate nvstrings, nvcategory and nvtext
```

Comment:
Changelog entry added twice..

Reply:
Thank you, saw just couple of CI jobs and assumed change-log is missing :)